R/palaeorotate.R

```diff
@@ -428,7 +428,7 @@ palaeorotate <- function(occdf, lng = "lng", lat = "lat", age = "age",
     # Define root
     pbase <- "https://gws.gplates.org/reconstruct/reconstruct_points/"
     # Setup query structure
-    query <- list(points = NULL, time = NULL,
+    query <- list(time = NULL, lons = NULL, lats = NULL,
```

Comment:
Is this due to an API change?

Reply:
Not really, the `points` argument is still available but I think this is cleaner for specifying coordinates and we don't need to mess around with generating the combined string....